### PR TITLE
BCF-2672 cleanup legacy job orm env var loading

### PR DESCRIPTION
**Pull request by ilija42**
- Rename some legacy env var loading functions to better represent what they do now.
- Remove PollPeriod value Override for VRF spec from job orm because it is already done un listenerV2 right before it is needed. 
- Remove direct request spec MinIncomingConfirmations value override from job orm as it is already done in delegate right before it is needed.
- Remove legacy variables that indicated whether certain job spec fields were overriden by env variable.

Operator-UI Changes:
https://github.com/smartcontractkit/operator-ui/pull/60

### CODE REVIEW

File: core/services/job/orm.go [hunk 1]
```diff
@@ -705,61 +705,41 @@ func (o *orm) FindJobs(offset, limit int) (jobs []Job, count int, err error) {
...
-			err = multierr.Combine(err, o.LoadEnvConfigVars(&jobs[i]))
+			err = multierr.Combine(err, o.LoadConfigVars(&jobs[i]))
 		}
 		return nil
 	})
 	return jobs, int(count), err
 }
 
-func (o *orm) LoadEnvConfigVars(jb *Job) error {
+func (o *orm) LoadConfigVars(jb *Job) error {
```
Comment (ilija42) [inline core/services/job/orm.go:715]: Does it make sense to remove this completely from ORM? This is also called in the delegate so all the functionality should be fine. IIUC only gql endpoint would be affected if this is removed.

**Comment by jinhoonbang** [comment]
VRF related changes look fine to me. One observation: `PollPeriodEnv` is not being used anywhere and can be deleted as follow up.

**Comment by ilija42** [comment]
> VRF related changes look fine to me. One observation: `PollPeriodEnv` is not being used anywhere and can be deleted as follow up.

I already deleted `PollPeriodEnv`, unless I missed it somewhere?

File: core/web/schema/type/spec.graphql
Comment (jmank88) [inline core/web/schema/type/spec.graphql:1]: Does this require an operator-ui update too?

Comment (ilija42) [reply core/web/schema/type/spec.graphql:1]: Yeah, just opened a PR there 
https://github.com/smartcontractkit/operator-ui/pull/60

**Comment by jinhoonbang** [comment]
> I already deleted PollPeriodEnv, unless I missed it somewhere?

I see that you deleted them. Thanks

**Review by jmank88 — APPROVED** [review]

**Pull request merged**